data/prod/c0000000027o.json

```diff
@@ -190,7 +190,7 @@
                 "blockId": "8fe17c4d-7138-eb79-90e6-135476782d87",
                 "type": "paragraph",
                 "model": {
-                  "text": "/Meghan's own bouquet was [placed on the tomb of the unknown warrior](/news/articles/c000000000po) at Westminster Abbey, following royal tradition."
+                  "text": "/Meghan's own bouquet was [placed on the tomb of the unknown warrior](/news/articles/c0000000027o) at Westminster Abbey, following royal tradition."
```

Comment:
Same comment as above, but with this linking to `/news/articles/c0000000025o` instead.